### ISSUE TITLE
Feature to search and  follow other members of the organization

### ISSUE THREAD

**Issue by harsh253** [codeuino/social-platform-donut-frontend#403]
**Problem**
Currently, in the donut platform we do not have the ability to search any member of the organization, neither can we follow them until and unless they have followed us before hand. Also, we can't view a member's stats like posts made my them, number of projects they are involved in, events attended by them etc. This is an important aspect of any social platform to promote interactivity among the members.
Following a member will give us the benefit of **viewing posts made by them** on our news feed section, **comment on their posts**, **view projects they are working on (non-private projects)** or if they are currently free to work on other projects then we can on board them to our projects (this is a long shot) , **view what events they are engaging in** which might be useful to us as well

**Solution**
1. So right now the organization tab from the sidebar has only 4 sub-sections, 'About Us', 'Donuts', 'Events', 'Projects' as shown below
![image](https://user-images.githubusercontent.com/32335320/78564881-b5196c80-783a-11ea-9123-7b61864f3367.png)
2. We can add one more sub-section, **'Members'**. 
3. Once into the members tab, there should be a **list of members** along with a **search functionality**
4. Once the member is found through search, the user can **follow** as well as **view their profile**.
5. If we decide to view their profile, we'll end up landing to a different page with profile info else if we decide to directly follow them, they'll receive a notification of the same and will get added to our following list.
5. Their profile will have stats like events attended, events created, their active projects, posts made by them , comments made by them , their tech stack and other info

@devesh-verma @vaibhavdaren @Rupeshiya @jaskirat2000 . I'd request you all to let me know your views on this so that I can proceed further with the development of these sections on frontend :)

**Comment by Rupeshiya** [comment]
@harsh253 Sounds good!
But instead of it should be an invitation button so that admin can directly invite from there itself.

**Comment by harsh253** [comment]
@Rupeshiya thanks for the feedback.
Here, I am considering that the members shown in the **Members list** are those users who have been already invited by the admin to join the community. 
Also I think that adding an invitation button in the platform doesn't add much value here rather it should be present in the admin panel, right ?

**Comment by Rupeshiya** [comment]
oh, @harsh253 I was thinking that if the user who logged in is an **admin** then there should be an **invite button**, else it should be **follow** button for the **other** members.

**Comment by harsh253** [comment]
I get your point @Rupeshiya . But the thing is that the admin portal will already be having the feature to invite users to community and that portal will be specifically be designed for purpose like these. So that's why I think , we should leave this feature (invite feature) specific to admin panel. 
Please let know your thoughts on this

**Comment by Rupeshiya** [comment]
I agreed @harsh253 but as this organization page will be common for both users as well as admin except some right which admin will have for that page. so I thought that.
@jaskirat2000 Please let us know your thoughts.

**Comment by harsh253** [comment]
Yup, you are right. Let's have @jaskirat2000 @vaibhavdaren and @devesh-verma's views on this as well once

**Comment by devesh-verma** [comment]
@harsh253 I need to think through this.

**Comment by harsh253** [comment]
All right @devesh-verma . Just let us know your view on this once you get time
